backend/schema.js

```diff
@@ -5,13 +5,15 @@ export default gql`
     id: ID!
     title: String!
     author: String!
+    pages: String
```

Comment:
Why is `pages` a string? Shouldn't it be an integer?
Can we call the attribute `numPages` for clarity?